### ISSUE TITLE
[MC 1.15.2] Bug Report Skulks Cannot Breed With Tall Fungi

### ISSUE THREAD

**Issue by Midnite-Moon** [Cryptic-Mushroom/The-Midnight#463]
# The Bug

When trying to breed, the skulk mob can’t be bred with all tall fungi variants from the Midnight. This includes nightshrooms, viridshrooms, dewshrooms, bogshrooms, and mistshrooms.

# Steps to reproduce

Simply place a variant of tall fungi in your hot bar and try feeding it to a nearby skulk.

# Important Notes

The game does not crash or cause any problems. It’s simply a consistency issue that needs to be patched.

## Environment

- Minecraft: [1.15.2]
- Midnight: [0.5.11]
- Forge: [31.2.47]
- No extra features, only Vanilla Minecraft plus the Midnight

**Comment by ShadewEnder** [comment]
Hm I'm inclined to think that is intentional but not sure... Such a little creature eating such a big fungus is kinda weird... I'll leave it open for now, maybe @Jonathing has some extra comments

**Comment by Jonathing** [comment]
Yes hi, I'm Jonathing. We'll probably talk in the dev server with what we'll make the skulk feedable with, since bagu added the feature on a whim without thinking about the other mushrooms. We'll probably just end up making a tag for those specific mushrooms, but we haven't discussed that in detail. Cheers.
